scss/_type.scss

```diff
@@ -1,27 +1,24 @@
-a {
-  &.tooltip-link {
-    > .material-icons {
-      color: $medium-gray;
-      vertical-align: middle;
+.tooltip-link {
```

Comment:
Here we drop the `a`

Reply:
https://stylelint.io/user-guide/rules/selector-no-qualifying-type